src/main/java/com/radixdlt/crypto/ECKeyPair.java

```diff
@@ -44,7 +44,7 @@
 /**
  * Asymmetric EC key pair provider fixed to curve 'secp256k1'.
  */
-@SecurityCritical({ SecurityKind.KEY_GENERATION, SecurityKind.SIG_SIGN, SecurityKind.PK_DECRYPT, SecurityKind.PK_ENCRYPT })
+@SecurityCritical({ SecurityKind.KEY_GENERATION, SecurityKind.KEY_STORE, SecurityKind.SIG_SIGN, SecurityKind.PK_DECRYPT, SecurityKind.PK_ENCRYPT })
```

Comment:
Would prefer if we removed the "fromFile" method altogether, which reads unencrypted keys.  This will be a security audit issue.

Reply:
@clstrfsck Either documentation of `KEY_STORE` is wrong, or you forgot what it states? :) documentation states: 

> Code **_stores_** private keys
So it is not about any keystore file, it is about classes having an instance variable representing a private key. 

And the `ECKeyPair` _stores_ a private key, but was not marked with `KEY_STORE` which is why I added it.

Reply:
Yes, the intention was that KEY_STORE was for secondary storage, ie disk.  Storing unencrypted keys on secondary storage was an issue raised during code audit, so we need to track it.  Wasn't intended for where a key is kept in memory.

Reply:
got it, fixed ✅